src/config/mod.rs

```diff
@@ -287,7 +287,7 @@ impl Config {
                 }
             }
 
-            // If none was found ther either, check in the user's configuration directory.
+            // If none was found there either, check in the user's configuration directory.
```

Comment:
the alternative is `here`, but this is the simplest correction, so it might be right...

Reply:
it's like that whoever typed it meant `there` so lets go with that